### PR TITLE
Resolve Issue #8: add i18n Support

### PR DESCRIPTION
**Pull request by mattb555**
Added functionality for i18n. Added English, German, and a Blank Template of current strings for further language integration. Added support for browser based language detection.

### CODE REVIEW

File: source/app.js [hunk 1]
```diff
@@ -132,6 +132,8 @@ class App extends React.PureComponent {
   }
 
   render () {
+    console.log(this.props.i18n);
```
Comment (xstable) [inline source/app.js]: please remove this comment.

File: source/i18n/en.json [hunk 2]
```diff
@@ -5,5 +5,10 @@
     "loadingText": "Data is loading...",
     "noDataText": "Currently no events",
     "pageText": "Page",
-    "ofText": "of"
+    "ofText": "of",
+    "date": "Date",
+    "event": "Event",
+    "postcode": "ZIP",
+    "place": "Adr",
```
Comment (xstable) [inline source/i18n/en.json]: please use "Location" instead of "Adr"

**Review by xstable — APPROVED** [review]

**Comment by mattb555** [comment]
I made the requested changes, and in addition I added a config file to the i18n folder to allow for toggling language auto detection.

**Comment by xstable** [comment]
@mattb555 Thanks a lot for your implementation

**Pull request merged**